terraform/README.md

```diff
@@ -0,0 +1,37 @@
+# Terraform files managing our organization
+
+## Setup
+
+1. Install terraform: `snap install terraform`
+1. [Install the GitHub CLI](https://github.com/cli/cli/blob/trunk/docs/install_linux.md)
+1. Login to the GitHub CLI and initialize terraform.
+
+   ```shell
+
+   cd tfcrafters
+   terraform init
+   gh auth cli
+   ```
+
+## Usage
+
+From the directory `tfcrafters`, execute the following commands.
```

Comment:
```suggestion
From the directory `terraform`, execute the following commands.
```

I think this was just an artifact from local dev, but really we probably only want this to run in CI so I'm not sure how helpful it is?